projects/plugins/boost/app/lib/minify/class-cleanup-stored-paths.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Lib\Minify;
+
+/**
+ * Takes care of cleaning up options created during concatenation.
+ *
+ * @since $$next-version$$
+ */
+class Cleanup_Stored_Paths {
+
+	/**
+	 * The maximum number of options to process in a single batch.
+	 *
+	 * @var int
+	 */
+	private $max_options_to_process = 50;
+
+	/**
+	 * The key of the option that stores the ID of the last processed option.
+	 *
+	 * @var string
+	 */
+	private $last_processed_option_key = 'jb_transient_cleanup_concat_paths_last_processed_option_id';
+
```

Comment:
Per offline discussion, let's prefix this with `jetpack_boost_` instead of `jb_transient_`

```suggestion
	private $last_processed_option_key = 'jetpack_boost_cleanup_concat_paths_last_processed_option_id';
```

Reply:
I didn't check the diff and just did a commit :D

3a42244225fe455fe658dc084205268003300997

Make sure you select the lines you want to change, so it removes the old one.

Reply:
Oh interesting! I used the VS Code GitHub PR plugin for the suggestion so I'm blaming that :D Will double check this in the future.

Reply:
No worries! GH actions caught it.